test/processors/base_processor_test.rb

```diff
@@ -184,9 +184,19 @@ def processor.ftp_upload_file(u,f,o); 'uploaded!'; end
     processor.original_format.must_equal 'ogg'
   end
 
-  it 'extracts format from uri' do
+  it 'extracts format from the uri' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.